Admin/FormAdmin.php

```diff
@@ -142,6 +146,16 @@ function (Localization $localization) {
                 ->addToolbarActions($formToolbarActions)
                 ->setParent(static::EDIT_FORM_ROUTE)
                 ->getRoute(),
+            $this->routeBuilderFactory->createListRouteBuilder(static::LIST_ROUTE_DATA, '/form_data')
+                ->setResourceKey('dynamics')
```

Comment:
Maybe `dynamic_forms`? If you have a lot of resource keys just `dynamics` does not tell a lot.